Node/test/TeamsChatConnector.js

```diff
@@ -444,4 +444,97 @@ describe('TeamsChatConnector', function () {
       })
     });
   });
-});
\ No newline at end of file
+
+  describe('#onComposeExtensionFetch()', function () {
```

Comment:
`#onComposeExtensionFetch()` to `#onComposeExtensionFetchTask()` for naming consistent

Reply:
fixed